scipy/optimize/__init__.py

```diff
@@ -255,12 +255,21 @@
 
    linprog_verbose_callback -- Sample callback function for linprog (simplex).
 
-Assignment problems:
+Assignment problems
+===================
 
 .. autosummary::
    :toctree: generated/
 
    linear_sum_assignment -- Solves the linear-sum assignment problem.
+   quadratic_assignment -- Approximates solution to the quadratic assignment problem and the graph matching problem.
```

Comment:
```suggestion
   quadratic_assignment -- Approximates solution to the quadratic assignment problem.
```
No sense in going over the line limit since this doesn't actually appear in the [`scipy.optimize` documentation](https://21182-1460385-gh.circle-artifacts.com/0/html-scipyorg/optimize.html).